admin-dev/themes/new-theme/package.json

```diff
@@ -5,80 +5,90 @@
   "private": "true",
   "scripts": {
     "test": "./node_modules/.bin/mocha  --require babel-register --reporter spec \"tests/**/*.spec.js\"",
-    "build": "NODE_ENV=production webpack --progress --colors --debug --display-chunks",
-    "dev": "webpack --progress --colors --debug --display-chunks --watch",
-    "watch": "npm run dev"
+    "build": "webpack --progress --colors --debug --display-chunks --mode=production",
+    "dev": "webpack --progress --colors --debug --display-chunks --watch --mode=development",
+    "watch": "npm run dev",
+    "lint": "eslint -c .eslintrc.js --ext .js,.vue .",
+    "lint-fix": "eslint -c .eslintrc.js --ext .js,.vue . --fix"
   },
   "author": "PrestaShop",
   "license": "OSL-3.0",
   "dependencies": {
-    "bootstrap-slider": "^6.0.16",
+    "bootstrap-slider": "^10.6.2",
     "bootstrap-tokenfield": "^0.12.0",
-    "bourbon": "^4.2.6",
+    "bourbon": "^6.0.0",
```

Comment:
Watch out, lots of things have been dropped in bourbon! This may be a breaking change.